src/mgmt/tests/mocks.c

```diff
@@ -421,6 +421,23 @@ void mock_ds_mgmt_cont_set_owner_teardown(void)
 	D_FREE(ds_mgmt_cont_set_owner_group);
 }
 
+int	ds_mgmt_exclude_return;
+uuid_t	ds_mgmt_exclude_uuid;
```

Comment:
nit - name alignment

Would be good for these variable names, and the name of the setup function, to more closely match "ds_mgmt_pool_target_update_state" since the function does not just exclude.